Translators/WZDx/main.py

```diff
@@ -105,7 +165,7 @@ def translateWzdxTIM():
         'Content-Type': 'application/json'
     }
 
-    tims = translate(request.get_json()["features"])
+    tims = translate_old(request.get_json()["features"])

```

Comment:
(non-blocking) What's the rationale for keeping this old functionality & how will the new functionality be invoked in comparison?

Reply:
The old function is currently being used by an exposed endpoint that allows users to directly pass a list of messages to the translator, instead of scraping the CDOT feed. However, at this time, we are only running the translator as a cron job. In the future, it may be worth reconsidering whether we still need this additional exposed endpoint.